### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t1821

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/7cfe1e18bfec0fd/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/7cfe1e18bfec0fd/output.txt

Total script time: 0.41 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/add26d98578522b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/add26d98578522b/output.txt

Total script time: 1.03 mins

Server is running on http://pr661.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/bfe129e9f8347d8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/bfe129e9f8347d8/output.txt

Total script time: 0.43 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/9a86f71f5810a56/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/9a86f71f5810a56/output.txt

Total script time: 1.23 mins

Server is running on http://pr661.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d44c0137a0d9b0e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d44c0137a0d9b0e/output.txt

Total script time: 0.49 mins

File: css/globals.less [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// GLOBALS
+//
```
Comment (secretrobotron) [inline css/globals.less]: Remove?

File: css/buttons.less [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/* BUTTONS
```
Comment (secretrobotron) [inline css/buttons.less]: Just curious about this comment title style. Where did you get it from? Is there something else we can use that looks like something we're already using?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fcdad0d92a966f0/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/fcdad0d92a966f0/output.txt

Total script time: 1.80 mins

Server is running on http://pr661.butyr.org/templates/test.html

File: css/buttons.less [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+.icon-grear-sign { background-position: -240px 0; }
+.icon-ok-sign { background-position: -72px -96px; }
+.icon-eye-open { background-position: -96px -120px; }
+.icon-share-alt { background-position: -336px -96px; }
+.icon-user { background-position: -168px 0; }
+.icon-plus-sign { background-position: -408px -96px; }
+.icon-minus { background-position: -433px -96px; }
+
+/* Specific Buttons
```
Comment (secretrobotron) [inline css/buttons.less]: Missed one!

File: css/tray-status-bar.less [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+// This is where the play/pause button is
+// TO-DO: prefix with butter-;
+.status-container {
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  top: 5px;
+  left: 0;
+  
+  .play-button-container {
```
Comment (secretrobotron) [inline css/tray-status-bar.less]: This is another duplicate, it seems. There are two places where `.play-button-container` is mentioned.

Comment (k88hudson) [inline css/tray-status-bar.less]: Damn, this whole section is a repeat.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b74e68334c2d46f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b74e68334c2d46f/output.txt

Total script time: 0.24 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3e3194380fcca00/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3e3194380fcca00/output.txt

Total script time: 1.08 mins

Server is running on http://pr661.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d9e37fdc5db6926/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/d9e37fdc5db6926/output.txt

Total script time: 0.18 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cad1e578ec4037f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cad1e578ec4037f/output.txt

Total script time: 1.20 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/69b145b65f9b97c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/69b145b65f9b97c/output.txt

Total script time: 0.62 mins

Server is running on http://pr661.butyr.org/templates/test.html

File: css/tray-timeline.less [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+    margin: 0;
+    height: 145px;
+  }
+  .tracks-container {
+    .butter-track {
+      width: 100%;
+      -moz-border-radius: 6px;
+      -o-border-radius: 6px;
+      -webkit-border-radius: 6px;
+      border-radius: 6px;
```
Comment (mjschranz) [inline css/tray-timeline.less]: Can't this just be the generic border-radius?

Then again, I think we have a minor ticket sitting around to handle any of these that are sitting around.

File: css/tray.less [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+  }
+
+  .fadable { 
+    //TO-DO: This should be moved into a general UI kit
+    .transition( opacity 0.5s );
+  }
+
+  /* Scrollbars */
+
+  // TO-DO: move to UI kit and prefix with butter-
```
Comment (mjschranz) [inline css/tray.less:181]: For all the TO-DOs that are listed at various areas of the code, is something already filed about it?

File: css/tray-timeline.less (hunk 5 above)
Comment (k88hudson) [inline css/tray-timeline.less]: good catch

File: css/tray.less (hunk 6 above)
Comment (k88hudson) [inline css/tray.less:181]: No, good point

Comment (k88hudson) [inline css/tray.less:181]: https://webmademovies.lighthouseapp.com/projects/65733-popcorn-maker/tickets/1841-convert-a-lot-of-over-specific-lesscss-to-namespacingprefixing

The other is already filed.

**Pull request merged**